packages/common/src/sites/_internal/computeProps.ts

```diff
@@ -42,6 +42,10 @@ export function computeProps(
   site.updatedDate = new Date(model.item.modified);
   site.updatedDateSource = "item.modified";
   site.isDiscussable = isDiscussable(site);
+
+  // set isHubHome
+  site.isHubHome = Boolean(model.item.properties?.isHubHome);
```

Comment:
Upon closer look, sites has it's own [property mapper](https://github.com/Esri/hub.js/blob/10379/hide-dashboard-pane/packages/common/src/sites/_internal/getPropertyMap.ts). Maybe it'd make more sense to add an entry there to facilitate this mapping rather than manually doing it here.

```ts
  map.push({
    entityKey: "isHubHome",
    storeKey: "item.properties.isHubHome",
  });
```

Though that wouldn't coerce `undefined` values to `false`, but I _think_ that's OK.